Userland/Utilities/yes.cpp

```diff
@@ -13,12 +13,22 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 {
     TRY(Core::System::pledge("stdio"));
 
-    StringView string = "yes"sv;
```

Comment:
1. The default should be `y`, not `yes`, right? (I realize the default predates your change.)
2. Coreutils `yes` accepts multiple strings, but BSD `yes` doesn't. Is there any benefit in accepting multiple strings? Does anything need this?

Reply:
1. Yes, the default should be `y`, not `yes`.
2. I guess you could use it with `xargs` easily if it can accept multiple strings.

Reply:
Do you have a concrete example where one would want to use yes with xargs?

Reply:
Nope.

Reply:
Then I'd suggest not merging this PR :)

One to change the default from `yes` to `y` would be cool though.

Reply:
Seems easiest, but you can also transmogrify the commit in this one and rename it. Up to you :)

Reply:
Nah, I'll make a new one. Thanks for reviewing it.